packages/labs/motion/package.json

```diff
@@ -61,7 +62,9 @@
     "typescript": "^4.3.5"
   },
   "dependencies": {
-    "lit": "^2.0.0"
+    "@lit/reactive-element": "^1.0.1",
+    "lit": "^2.0.0",
```

Comment:
The thought was that SpringController only needs ReactiveElement, though as a package dependency this doesn't help since we're already depending on `lit`.